pkg/ui/app/components/dropdown.tsx

```diff
@@ -20,8 +31,10 @@ export default class Selector extends React.Component<SelectorOwnProps, {}> {
   render() {
     let {selected, options, onChange} = this.props;
     return <div className="dropdown">
-      <span className="dropdown__title">{ this.props.title }:</span>
+      <span className="Select__side-arrow" dangerouslySetInnerHTML={trustIcon(leftArrow)} onClick={() => this.props.onArrowClick(ArrowDirection.LEFT)}></span>
```

Comment:
This should be `dropdown__side-arrow`?  This is not inside of the `Select` element.

I'm not thrilled about the arrows being present on all dropdowns, but you put the relevant CSS in the shame file so I guess that's fine for now.  Maybe a TODO here?